src/docs/options.md

```diff
@@ -55,58 +55,87 @@ levels in different tasks.
 Option Types
 ------------
 
-Option value literal formats are the same on the command-line, environment variables and in `pants.ini`,
-with a couple of extra command-line conveniences.
+Option value formats are the same on the command-line, environment variables and in `pants.toml`, except for list options. There are also a couple of extra command-line conveniences.
 
 ### Boolean Options
 
-The boolean option values are: `false`, `False`, `true`, `True`.
+The boolean option values are `false` and `true`.
```

Comment:
Technically, you can still use `False` and `True` with the command line and env vars. But, consolidating around `false` and `true` aligns us with the config file and simplifies things for users.